src/OpenLoco/Objects/ObjectManager.cpp

```diff
@@ -33,7 +33,7 @@ namespace OpenLoco::ObjectManager
         Object** objects;
         ObjectEntry2* object_entry_extendeds;
     };
-    static_assert(sizeof(ObjectRepositoryItem) == 8);
+//    static_assert(sizeof(ObjectRepositoryItem) == 8);
```

Comment:
Can you create a new assert that is 32bit only it would be very annoying to lose these